include/xtensor/xassign.hpp

```diff
@@ -695,8 +773,8 @@ namespace xt
     template <class E1, class E2>
     inline void linear_assigner<false>::run(E1& e1, const E2& e2)
     {
-        using is_convertible = std::is_convertible<typename std::decay_t<E2>::value_type,
-                                                   typename std::decay_t<E1>::value_type>;
+        using is_convertible = std::
+            is_convertible<typename std::decay_t<E2>::value_type, typename std::decay_t<E1>::value_type>;
```

Comment:
```suggestion
        using is_convertible = std::is_convertible<
            typename std::decay_t<E2>::value_type, 
            typename std::decay_t<E1>::value_type>;
```
My guess is that clang tries to align:
```cpp
        using is_convertible = std::is_convertible<typename std::decay_t<E2>::value_type, 
                                                   typename std::decay_t<E1>::value_type>;
```
but that the line is then still too long. Personally, I am not using that alignment mode: it requires several lines to be reformatted if the first line changes